examples/deeplab/src/DrawOutputs.js

```diff
@@ -0,0 +1,886 @@
+class Renderer {
+  constructor(canvas) {
+
+    this.gl = canvas.getContext('webgl2');
+    if (this.gl === null) {
+      throw new Error('Unable to initialize WebGL.');
+    }
+
+    this.guidedFilter = new GuidedFilter(this.gl);
+    this.utils = new WebGLUtils(this.gl);
+
+    this.shaders = {};
+
+    this._segMap = null;
+    this._predictions = null;
+    this._clippedSize = [224, 224];
+    this._imageSource = null;
+
+    // workaround for lack of `align_corner` in `RESIZE_BILINEAR`
+    this._correctionFactor = 0.99;
```

Comment:
Please remove this workaround and file an issue to support `align_corner` in WebNN API.